src/Symfony/Component/JsonStreamer/Mapping/PropertyMetadata.php

```diff
@@ -73,9 +73,7 @@ public function withNativeToStreamValueTransformers(array $nativeToStreamValueTr
     public function withAdditionalNativeToStreamValueTransformer(string|\Closure $nativeToStreamValueTransformer): self
     {
         $nativeToStreamValueTransformers = $this->nativeToStreamValueTransformers;
-
         $nativeToStreamValueTransformers[] = $nativeToStreamValueTransformer;
-        $nativeToStreamValueTransformers = array_values(array_unique($nativeToStreamValueTransformers));
```

Comment:
Stop trying to make value transformers unique, as it does not add any value, and didn't work anyway because `\Closure` aren't castable to `string`.